src/server/scripts/Northrend/Ulduar/Ulduar/boss_flame_leviathan.cpp

```diff
@@ -675,6 +644,9 @@ void boss_flame_leviathan::boss_flame_leviathanAI::KilledUnit(Unit* who)
 {
     if (who == me->GetVictim())
         events.RescheduleEvent(EVENT_PURSUE, 0);
+
+    if (Player* plr = who->ToPlayer())
```

Comment:
@Nefertumm here you are declaring the `plr` variable but then never using it. That's why Travis is complaining.

Reply:
Oh, i didn't see. I forgot to change it.